shared/wallets/send-form/participants/to-field.js

```diff
@@ -6,222 +6,211 @@ import {ParticipantsRow} from '../../common'
+          <Kb.Button
+            type="Wallet"
+            style={styles.createNewAccountButton}
+            label="Create a new account"
+            onClick={this.props.onCreateNewAccount}
           />
-        )
-      }
-    } else if (this.props.recipientType === 'stellarPublicKey') {
-      // Case #4: A user is sending to a stellar address that is either not associated to a keybase user or not complete. Show input for a stellar address.
-      component = (
-        <Kb.Box2 direction="vertical" fullWidth={true} style={styles.inputBox}>
-          <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.inputInner}>
-            <Kb.Icon
-              type={
-                this.props.incorrect || this.props.toFieldInput.length === 0
-                  ? 'icon-stellar-logo-grey-16'
-                  : 'icon-stellar-logo-16'
-              }
-              style={Kb.iconCastPlatformStyles(styles.stellarIcon)}
-            />
-            <Kb.NewInput
-              type="text"
-              onChangeText={this.props.onChangeRecipient}
-              textType="BodySemibold"
-              placeholder={'Stellar address'}
-              placeholderColor={Styles.globalColors.black_20}
-              hideBorder={true}
-              containerStyle={styles.input}
-              multiline={true}
-              rowsMin={2}
-              rowsMax={3}
-            />
-          </Kb.Box2>
-          {!!this.props.incorrect && (
-            <Kb.Text type="BodySmall" style={styles.errorText}>
-              {this.props.incorrect}
-            </Kb.Text>
-          )}
         </Kb.Box2>
       )
-    } else if (this.props.onShowProfile) {
-      // Case #5: A user is sending to a keybase user but has not selected a keybase user yet. Show search input for keybase users.
-      return (
-        <Search
-          onClickResult={this.onSelectRecipient}
-          onClose={() => {}}
-          onShowSuggestions={this.props.onShowSuggestions}
-          onShowTracker={this.props.onShowProfile}
-        />
-      )
+    }
+
+    // A user is sending from an account to another account with other
+    // accounts. Show a dropdown list of other accounts, in addition
+    // to the link existing and create new actions.
+    let items = [
+      <DropdownText key="link-existing" text="Link an existing Stellar account" />,
+      <DropdownText key="create-new" text="Create a new account" />,
+    ]
+
+    if (this.props.allAccounts.length > 0) {
+      const walletItems = this.props.allAccounts.map(account => (
+        <DropdownEntry key={account.id} account={account} user={this.props.user} />
+      ))
+      items = walletItems.concat(items)
     }
 
     return (
-      <ParticipantsRow
-        heading="To"
-        headingAlignment={this.props.recipientType === 'otherAccount' ? 'Right' : 'Left'}
-        headingStyle={
-          this.props.recipientType === 'stellarPublicKey' && !this.props.recipientUsername
```

Comment:
removed this logic, since we're not converting public keys to usernames anymore, right?